engine/events/specials.asm

```diff
@@ -25,6 +25,7 @@ Special_GameCornerPrizeMonCheckDex:
 	ret nz
 	call SetSeenAndCaughtMon
 	call FadeToMenu
+	call BackupSprites
```

Comment:
All of the specials call `FadeToMenu` and then `BackupSprites1. So home/clear_sprites.asm could be updated:
```asm
FadeToMenu_BackupSprites::
	call FadeToMenu
BackupSprites::
```
and then just have one `call FadeToMenu_BackupSprites` here.